apis/nft/ts/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@thecointech/nft",
+  "version": "0.2.5",
+  "description": "OpenAPI client for @thecointech/nft",
+  "author": "OpenAPI-Generator Contributors",
+  "keywords": [
+    "axios",
+    "typescript",
+    "openapi-client",
+    "openapi-generator",
+    "@thecointech/nft"
+  ],
+  "license": "Unlicense",
```

Comment:
This is the file I think that broke your build the other day.  Was accidentally ignored

Reply:
Remind me of this on Monday.  I'm not sure what you mean, but it obvs needs to be fixed!